libs/ui/src/lib/facets/facet-block/facet-block.component.ts

```diff
@@ -40,7 +70,7 @@ export class FacetBlockComponent implements OnInit {
   }
 
   onFilterChange(value: string) {
-    this.filterChange.emit(value)
+    this.filterChange.emit(`.*${value}*`)
```

Comment:
Shouldn't this be done at the smart container level ? The UI components are not aware of business logic, just user interactions.

Reply:
You are perfectly right ! What a mistake ;)